test/src/unit-cppapi-vfs.cc

```diff
@@ -167,3 +167,69 @@ TEST_CASE(
   // Clean up
   vfs.remove_dir(path);
 }
+
+TEST_CASE(
+    "C++ API: Test VFS copy file",
+    "[cppapi], [cppapi-vfs], [cppapi-vfs-copy-file]") {
+  using namespace tiledb;
+  Context ctx;
+  VFS vfs(ctx);
+
+#ifdef _WIN32
+  std::string path = sm::Win::current_dir() + "\\vfs_test\\";
```

Comment:
CI might complain about an unused variable here. I would wrap this entire test in `#ifndef _WIN32`.